libs/core/src/lib/timeline/components/timeline-node-footer/timeline-node-footer.component.ts

```diff
@@ -0,0 +1,10 @@
+import { Component } from '@angular/core';
+
+@Component({
+  selector: 'fd-timeline-node-footer',
+  templateUrl: './timeline-node-footer.component.html',
```

Comment:
Can we add encapsulation and changeDetection?

Reply:
Please use `template` instead of templateUrl.